src/AutoRest.CSharp/Common/Generation/Writers/JsonCodeWriterExtensions.cs

```diff
@@ -246,7 +246,7 @@ private static void ToSerializeCall(this CodeWriter writer, FormattableString wr
                 if (property.ValueSerialization == null)
                 {
                     // Flattened property
-                    writer.Line($"{writerName}.WritePropertyName({property.SerializedName:L});");
```

Comment:
Maybe we can change here to append `u8`? https://github.com/Azure/autorest.csharp/blob/b8ebef729c64db45fe033830699ccb4ac0fe425a/src/AutoRest.CSharp/Common/Generation/Writers/CodeWriter.cs#L385